### PR TITLE
cleanup event in execution_controller

### PR DESCRIPTION
**Pull request by Garrybest**
Signed-off-by: Garrybest <garrybest@foxmail.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Some resource's scope is `Cluster`, so we use `klog.KObj`

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controllers/execution/execution_controller.go [hunk 1]
```diff
@@ -292,11 +292,11 @@ func (c *Controller) updateAppliedCondition(work *workv1alpha1.Work, status meta
 	})
 }
 
-func (c *Controller) eventf(object *unstructured.Unstructured, eventtype, reason, messageFmt string, args ...interface{}) {
+func (c *Controller) eventf(object *unstructured.Unstructured, eventType, reason, messageFmt string, args ...interface{}) {
 	ref, err := helper.GenEventRef(object)
 	if err != nil {
-		klog.Errorf("ignore event(%s) as failed to build event reference for: kind=%s, %s/%s due to %v", reason, object.GetKind(), object.GetNamespace(), object.GetName(), err)
+		klog.Errorf("ignore event(%s) as failed to build event reference for: kind=%s, %s due to %v", reason, object.GetKind(), klog.KObj(object), err)
```
Comment (duanmengkk) [inline pkg/controllers/execution/execution_controller.go:298]: Should we do the same way in other places?

Comment (XiShanYongYe-Chang) [reply pkg/controllers/execution/execution_controller.go:298]: I have the same doubts and often see this message in the log: `(/xxx) `. Perhaps we can create a work issue that records the modification of similar problems in all code modules.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

File: pkg/controllers/execution/execution_controller.go (hunk 1 above)
Comment (RainbowMango) [reply pkg/controllers/execution/execution_controller.go:298]: Good question.
I have an idea in my mind to implement one function for generating the `key` just for logging.
I'll try to do this and make a benchmark with `Klog`.

**Pull request merged**